README.md

```diff
@@ -1,149 +1,62 @@
-| IAR      | VRRRPPP (V = Version; RRR = Revision; PPP = Patch)   |
+## Application functionality
 
-#### CPUID Register Information
+The `main()` function is the single thread in the application, it toggles the state of a digital output connected to an LED on the target.
 
-| Bit Field | Field Description | Values |
-| --------- | ----------------- | ------ |
-|[31:24]    | Implementer       | 0x41 = ARM |
-|[23:20]    | Variant           | Major revision 0x0  =  Revision 0 |
-|[19:16]    | Architecture      | 0xC  = Baseline Architecture |
-|           |                   | 0xF  = Constant (Mainline Architecture) |
-|[15:4]     | Part Number       | 0xC20 =  Cortex-M0 |
-|           |                   | 0xC60 = Cortex-M0+ |
-|           |                   | 0xC23 = Cortex-M3  |
-|           |                   | 0xC24 = Cortex-M4  |
-|           |                   | 0xC27 = Cortex-M7  |
-|           |                   | 0xD20 = Cortex-M23 |
-|           |                   | 0xD21 = Cortex-M33 |
-|[3:0]      | Revision          | Minor revision: 0x1 = Patch 1 |
+## Building and Running
 
+1. Connect a USB cable between the USB port on the target and the host computer.
+2. Run the following command to build the example project and program the microcontroller flash memory:
+    ```bash
+    $ mbed compile -m <TARGET> -t <TOOLCHAIN> --flash
+    ```
+The binary is located at `./BUILD/<TARGET>/<TOOLCHAIN>/mbed-os-example-blinky.bin` and can alternatively be manually copied to the target which gets mounted on the host computer via USB.
 
+## Bare metal
+The application above is built with the full Mbed OS library including its RTOS components. However, for single-threaded applications running on targets with ultraconstrains, it is possible to obtain an application with an even smaller memory footprint using [Mbed OS bare metal](https://os.mbed.com/docs/mbed-os/latest/reference/mbed-os-bare-metal.html) profile.
+An application configuration file, [`config_bare_metal.json`](./config_bare_metal.json) is provided to build a binary with Mbed OS bare metal profile.
 
-You can view individual examples and additional API information of the statistics collection tools at the bottom of the page in the [related links section](#related-links).
+1. Connect a USB cable between the USB port on the target and the host computer.
+2. Run the following command to build the example project with runtime statistics output:
+    ```bash
+    $ mbed compile -m <TARGET> -t <TOOLCHAIN> --app-config config_bare_metal.json --flash
+    ```
+The binary is located at `./BUILD/<TARGET>/<TOOLCHAIN>/mbed-os-example-blinky.bin` and can alternatively be manually copied to the target which gets mounted on the host computer via USB.
 
+`"target.default_lib" : "small"` tells the build tool to use a small implementation of the C standard library for the toolchain selected if available. That would be Newlib-nano and MicroLib for GCC_ARM and ARM toolchains respectively.
```

Comment:
I would rather see this information in mbed-os-bare-metal.html when the feature is ready.

Reply:
[Done](https://github.com/ARMmbed/mbed-os-example-blinky/pull/192/commits/2f96047ab8eb914c0ce2ee6805a598a1df3445e4).